stories/Text/index.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
```

Comment:
Convert ur story to the auto example one. 
you can see an example in Autocomplete

Reply:
I wrote is as a todo on top :)
Creating a story on a seperated pr for `Heading`, `UIText` & `Text` because on each one there is a different issue to solve, and I don't want the story to block it

Reply:
creating separated ones
UIText: https://github.com/wix/wix-ui-backoffice/pull/77
Heading: https://github.com/wix/wix-ui-backoffice/pull/76
Text: after this will get merged